### PR TITLE
SLO1 performance issues

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: content/issues/2023-03-09-slo1-error-budget.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - Workers
+  - Dashboard
+resolved: false
+section: issue
+severity: notice
+---
+
+We are having SLO1 error budget issues.
+
+This are mostly caused by performance issues and we are working on them and planning more improvements.
```
Comment (lachmanfrantisek) [inline content/issues/2023-03-09-slo1-error-budget.md:15]:
```suggestion
These are mostly caused by performance issues and we are working on them and planning more improvements.
```

Comment (lachmanfrantisek) [reply content/issues/2023-03-09-slo1-error-budget.md:15]: Maybe, we can link the relevant issue(s).

Comment (mfocko) [inline content/issues/2023-03-09-slo1-error-budget.md:15]:
```suggestion
These are mostly caused by performance issues and we are working on them and planning more improvements.
```

**Review by mfocko — COMMENTED** [review]
otherwise LGTM

File: content/issues/2023-03-09-slo1-error-budget.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+affected:
+  - API
+  - Workers
+  - Dashboard
+resolved: false
+section: issue
+severity: notice
+---
+
+We are having SLO1 error budget issues.
```
Comment (lachmanfrantisek) [inline content/issues/2023-03-09-slo1-error-budget.md:13]: Can we link what SLO1 means?

Comment (lachmanfrantisek) [reply content/issues/2023-03-09-slo1-error-budget.md:13]: Here it is: https://packit.dev/docs/service-level-objectives/

File: content/issues/2023-03-09-slo1-error-budget.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+These are mostly caused by performance issues and we are working on them and planning more improvements.
```
Comment (lbarcziova) [inline content/issues/2023-03-09-slo1-error-budget.md:15]: Should we mention that we will prioritise fixing the SLO-related issues over working on other features?

**Pull request merged**